website/sidebars.json

```diff
@@ -1,194 +1,173 @@
-                    "integrator-section/verifiable-document/did/signing-document",
-                    "integrator-section/verifiable-document/did/revoking-document-did"
-                  ]
-                },
-                "integrator-section/verifiable-document/differences",
-                "integrator-section/verifiable-document/going-further"
-              ]
-            },
-            {
-              "type": "category",
-              "label": "Transferable Record",
-              "items": [
-                "integrator-section/transferable-record/overview-tr",
-                "integrator-section/transferable-record/token-registry",
-                "integrator-section/transferable-record/preparing-transferable-record",
-                "integrator-section/transferable-record/issuing-transferable-record",
-                "integrator-section/transferable-record/going-further-tr"
-              ]
-            }
-          ]
-        },
-        {
-          "type": "category",
-          "label": "Web App Tutorial",
-          "items": [
-            "integrator-section/webapp-tutorial/deploy-document-store-webapp",
-            "integrator-section/webapp-tutorial/issue-document-store-webapp",
-            "integrator-section/webapp-tutorial/revoke-document-store-webapp"
-          ]
-        }
-      ]
-    }
+
+  "revoke": [
+      "revoke-section/revoke",
+      "revoke-section/diff-btw-revocation-methods"
   ],
-  "developer": [
-    {
-      "type": "category",
-      "label": "Quickstart (Code)",
-      "items": [
-        "developer-section/quickstart/create-verifiable-document-issuer",
-        "developer-section/quickstart/create-custom-renderer",
-        "developer-section/quickstart/create-custom-schema",
-        "developer-section/quickstart/configure-dns",
-        "developer-section/quickstart/oa-embedded-html",
-        "developer-section/quickstart/oa-embedded-qrcode"
-      ]
-    },
-    {
+
+  "verify": [
+    "verify-section/verify",
+    "verify-section/main-properties",
+    "verify-section/document-integrity",
+    "verify-section/issuance-status",
+    "verify-section/issuance-identity"
+  ],
+
+  "distribute": [
+    "distribute-section/distribute",
+    "distribute-section/oa-embed",
+    "distribute-section/oa-embed-html",
+    "distribute-section/oa-embed-qrcode"
+  ],
+
+  "lib": [
+        "lib-section/overview-lib",
+        "lib-section/remote-files/open-attestation",
+        "lib-section/remote-files/open-attestation-cli",
+        "lib-section/remote-files/open-attestation-encryption",
+        "lib-section/remote-files/open-attestation-verify",
+        "lib-section/remote-files/document-store",
+        "lib-section/remote-files/token-registry",
+        "lib-section/remote-files/decentralized-renderer-react-components",
+        "lib-section/remote-files/decentralized-renderer-react-template"
+  ],
+
+  "guides": [
```

Comment:
should we add an `overview` article here? Not sure if we should add one for FAQ as well

Reply:
For completeness, it will be good to have the `overview` article in both places.